rdmo/projects/viewsets.py

```diff
@@ -415,6 +420,119 @@ def hierarchy(self, request, pk):
         serializer = ProjectHierarchySerializer(cached_trees[0], context=serializer_context)
         return Response(serializer.data)
 
+    @action(detail=True, methods=['get'], permission_classes=(HasModelPermission | HasProjectPermission, ))
+    def answers(self, request, pk, snapshot_id=None):
```

Comment:
so this action can still accept a snapshot but only via  `answers_snapshot`??
Is the `answers_snapshot` there only for the tests?